### PR TITLE
Adding Skeletion file for Pasadena

### PR DESCRIPTION
**Pull request by vickymyers**
This file will be for things to do in Pasadena @loriNakamura 
👸

### CODE REVIEW

File: vick-calf.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ # Pasadena
+
```
Comment (githubteacher) [inline vick-calf.md:2]: Perfect, that's where we are!

File: vick-calf.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+ ## Places to Eat
+
```
Comment (githubteacher) [inline vick-calf.md:4]: Good, cause I'm hungry! Is there a good taco place?

File: vick-calf.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+ ###place to vistit
+
```
Comment (githubteacher) [inline vick-calf.md:6]: Any live music?

**Review by githubteacher — CHANGES_REQUESTED** [review]

File: vick-calf.md (hunk 3 above)
Comment (vickymyers) [reply vick-calf.md:6]: Phil Collins in Paris

**Review by githubteacher — APPROVED** [review]

**Pull request merged**